CMakeLists.txt

```diff
@@ -42,6 +42,15 @@ set(ANTLR4_WITH_STATIC_CRT OFF)
 set(ANTLR4_TAG tags/4.9.3)
 include(ExternalAntlr4Cpp)
 
+FIND_PACKAGE(LibXml2)
+IF (WIN32)
```

Comment:
Doing this on a WIN32 basis will work fine for our internal build purposes, but in the field a given box may or may not have libxml2 on it already, so a search (probably using `find_package`) will make more sense. 

I thought I had seen some messages that we were already doing that. Was that in a different draft/branch?

Reply:
Yes, that was a different branch. I think I put this back at the time to minimize complexity, but I can put it back now. Thanks!